build.rs

```diff
@@ -1,8 +1,54 @@
-extern crate cc;
+use color_eyre::{
+    eyre::{eyre, Context, Report, Result},
+    Help, SectionExt,
+};
+use std::{env, fs, path::PathBuf, process::Command};
 
-use std::env;
+trait CommandExt {
```

Comment:
I can't find where these extension methods are used. It looks like the last usage might have been removed in 412829b742c86ced624270a107cb74a14cc4cad0.